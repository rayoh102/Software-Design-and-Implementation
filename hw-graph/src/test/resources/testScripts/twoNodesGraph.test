# Tests adding two nodes to the graph

# Create a graph
CreateGraph twoNodesGraph

# Add two nodes
AddNode twoNodesGraph a
AddNode twoNodesGraph b

# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
ListNodes twoNodesGraph
ListChildren twoNodesGraph a
ListChildren twoNodesGraph b