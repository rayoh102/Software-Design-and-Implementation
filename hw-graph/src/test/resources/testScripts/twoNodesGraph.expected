# Tests adding two nodes to the graph

# Create a graph
created graph twoNodesGraph

# Add two nodes
added node a to twoNodesGraph
added node b to twoNodesGraph

# Print the nodes in the graph, the outgoing edges from a and the outgoing edges from b
the children of a in twoNodesGraph are:
the children of b in twoNodesGraph are: